packages/aws-cdk-lib/aws-appsync/README.md

```diff
@@ -232,6 +232,83 @@ httpDs.createResolver('MutationCallStepFunctionResolver', {
 });
 ```
 
+### EventBridge
```

Comment:
Can we add 1 or 2 sentences saying when using event bridge + appsync is beneficial. A general use case etc?

We are probably missing these for other integrations but still a good thing to add.